src/operator/random/multisample_op.h

```diff
@@ -67,7 +67,7 @@ inline bool MultiSampleOpShape(const nnvm::NodeAttrs& attrs,
   const MultiSampleParam& param = nnvm::get<MultiSampleParam>(attrs.parsed);
   mxnet::TShape sshape = param.shape;
   for (int i = 0; i < sshape.ndim(); ++i) {
-    CHECK_GT(sshape[i], 0) << "shape parameter must be non-zero within each dimension";
+    CHECK_GE(sshape[i], 0) << "shape parameter must be non-zero within each dimension";
```

Comment:
now the error message and the check don't align

Reply:
Got that